dev/ci/docker_linux/Dockerfile

```diff
@@ -122,5 +125,13 @@ RUN gem install bundler -N
 COPY ci/docker_linux/Gemfile /Gemfile
 COPY ci/docker_linux/Gemfile.lock /Gemfile.lock
 
+# Install dependencies for desktop flutter run
+RUN apt-get install -y --no-install-recommends \
```

Comment:
can these not be part of the earlier `apt-get install`?

Reply:
They can, I just wanted to build the Dockerfile progressively so that it was clear that the previous steps didn't need these packages.

Reply:
that makes sense. just FYI this may lead to a bigger overall image size (although maybe with caching image size isn't an issue for our cirrus builds?)

Reply:
Why would it be bigger? Just the layer overhead on the Docker image? In the end, the same files are installed.

Reply:
I guess we're never doing `apt-get clean`, so it probably wouldn't get bigger. Ideally we would use the minimum required install layers, and `apt-get clean` at the end of each.

Reply:
OK, I added two calls to `apt-get clean`.

I think we could do a lot to minimize layers if we wanted to: there are a lot of lines that could be run as continuation lines to keep them in one layer, but do we know how much that would really save? It would mean that failures would cause more of the image to be rebuilt each time (but we rebuild really infrequently, so that's not a huge deal).

Reply:
Thanks for your change. I think it's probably not a big deal, as I think the whole image gets cached. Everything looks good to me except for @CaseyHillers's comments regarding CIPD. I think we should try to land that in a different change, however, as the permissions would need to be figured out.